dashboard/starter-example/app/ui/dashboard/nav-links.tsx

```diff
@@ -26,9 +25,7 @@ export default function NavLinks() {
           <a
             key={link.name}
             href={link.href}
-            className={clsx(
-              'flex h-[48px] grow items-center justify-center gap-2 rounded-md bg-gray-50 p-3 text-sm font-medium hover:bg-sky-100 hover:text-blue-600 md:flex-none md:justify-start md:p-2 md:px-3',
-            )}
```

Comment:
Just a heads up that removing `clsx` from this file has "broken" the quiz at nextjs.com/learn - the question is now stale and the responses don't make sense.


<img width="691" alt="CleanShot 2023-10-30 at 14 34 26@2x" src="https://github.com/vercel/next-learn/assets/425059/7e7a24e5-ac02-4c11-be06-bcce4ee6f62d">